### PR TITLE
coordinateSystem is now private in Image

### PR DESCRIPTION
**Pull request by sebastianbaginski**
Fixes https://github.com/cogneco/ooc-kean/issues/1029
@emilwestergren is this what you meant in this issue ? also review

### CODE REVIEW

File: test/draw/RasterBgrTest.ooc [hunk 1]
```diff
@@ -77,28 +77,28 @@ RasterBgrTest: class extends Fixture {
...
+			image2 _coordinateSystem = CoordinateSystem Default
 			image2 canvas draw(image)
 			for (row in 0 .. image2 height)
 				for (column in 0 .. image2 width)
 					expect(image2[column, row] == image[image width - column - 1, row])
 
-			image coordinateSystem = CoordinateSystem Default
-			image2 coordinateSystem = CoordinateSystem XRightward | CoordinateSystem YDownward
+			image _coordinateSystem = CoordinateSystem Default
+			image2 _coordinateSystem = CoordinateSystem XRightward | CoordinateSystem YDownward
```
Comment (sebastianbaginski) [inline test/draw/RasterBgrTest.ooc]: I will update the tests later

**Comment by emilwestergren** [comment]
Yes. @ziquanyu has been working with this as well, are you in sync with each other?

**Comment by sebastianbaginski** [comment]
I saw his PRs, they were related to GPU images, I think he did not change the base class as here.

**Comment by emilwestergren** [comment]
The next step is to fix all the illegal accesses to `_coordinateSystem`.

**Comment by sebastianbaginski** [comment]
Yes, I will start here https://github.com/cogneco/ooc-kean/issues/1046

**Comment by ziquanyu** [comment]
@sebastianbaginski Sorry, I forgot to update to you last week after the meeting with Simon. These changes were actually done last week. I didn't make PR, but carried on. It's fine. I will soon update how's it going now about this Coordinate system changes.

**Comment by sebastianbaginski** [comment]
Alright thanks. @marcusnaslund : @ziquanyu told me that it is ok to merge this one, as he has done exactly the same changes and he can rebase his code

**Comment by marcusnaslund** [comment]
Thank you. Will be merged soon.

**Pull request merged**